xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -744,10 +744,9 @@ namespace XBMCAddon
     /// | xbmc.SERVER_EVENTSERVER   | [Set eventServer part that accepts remote device input on all platforms](http://kodi.wiki/view/EventServer)
     /// | xbmc.SERVER_ZEROCONF      | [Control Kodi's Avahi Zeroconf](http://kodi.wiki/view/Zeroconf)
     /// @param bStart               bool - start (True) or stop (False) a server
-    /// @param bWait                [opt] bool - wait on stop before returning (not supported by all servers)
     /// @return                     bool - True or False
     ///
-    ///
+    /// @python_v20 Removed option **bWait**.
```

Comment:
Please keep the 2 empty lines (as before) and move this new line below the thematic break (---). Keep an empty line before the example (as before). Yeah it sounds silly but doxygen breaks easily :)
Thanks for the improvement!

Reply:
Updated thanks!